.github/workflows/release-pr.yml

```diff
@@ -61,7 +61,7 @@ jobs:
 
       - name: "Generate the changelog"
         id: generate_changelog
-        uses: ./.github/actions/generate-changelog
+        uses: ./.github/actions/process-changelog
```

Comment:
Should we have a retry for failure for these? I did get an email from GH for the failure but I am trying to understand what's the ideal way for the release lead to know something isn't right in case of a failure and then acoid proceeding without noticing this.

Reply:
I don't think we need to have that.

`release-pr` should run once every 3 weeks just before the final week during which we release WCPay. I don't see any reason for it to fail at this step unless there isn't any changelog to generate (very unlikely after 3 weeks without releasing). 
Moreover, the release lead has to ask for a review for the PR from the release branch targeting trunk after that, and I'm confident it would be caught if neither the changelog nor the version (the step after that) is bumped across multiple files. The [doc](https://href.li/?https://fieldguide.automattic.com/developing-woocommerce-payments/client-deployment#prepare-release) (point 4) explicitly says to check for this when looking at the PR (which, by the way, would not be created automatically by the workflow in this case since this is the last step, so this alone should also make the release lead aware that something's weird).